lib/src/widgets/animated_button.dart

```diff
@@ -130,6 +132,42 @@ class _AnimatedButtonState extends State<AnimatedButton>
     }
   }
 
+  /// sets width and size animation
+  void _updateWidth() {
+    final theme = Theme.of(context);
+    final fontSize = theme.textTheme.button.fontSize;
+
+    var renderParagraph = RenderParagraph(
+      TextSpan(
+        text: widget.text,
+        style: TextStyle(
+          fontSize: fontSize,
+          fontWeight: theme.textTheme.button.fontWeight,
+          letterSpacing: theme.textTheme.button.letterSpacing,
+        ),
+      ),
+      textDirection: TextDirection.ltr,
+      maxLines: 1,
+    );
+
+    renderParagraph.layout(BoxConstraints(minWidth: 120.0));
+
+    // text width based on fontSize, plus 64.0 for padding
+    var textWidth =
+        renderParagraph.getMinIntrinsicWidth(fontSize).ceilToDouble() + 64.0;
```

Comment:
64.0 padding seems it might be a little too much for lower resolution screens? Here's an example:
<img width="356" alt="Screen Shot 2019-12-04 at 10 58 03 PM" src="https://user-images.githubusercontent.com/55109813/70206528-78ebc900-16ed-11ea-9a1d-be24e9e77921.png">